### ISSUE TITLE
CONTROLLERS: Verify User by Id

### ISSUE THREAD

**Issue by ElbekDeveloper** [ElbekDeveloper/Tarteeb#330]
# The Ask
Create an endpoint that verifies the user by the provided id

**Comment by ElbekDeveloper** [comment]
It is partially done. Once #329 is finished, update the endpoint to handle exceptions that service throws

**Comment by Nodirkhan** [comment]
I took this

**Comment by devdoston** [comment]
It's already done by @Nodirkhan
